client/state/themes/selectors/get-canonical-theme.js

```diff
@@ -39,7 +39,7 @@ export const knownConflictingThemes = new Set( [ 'bistro' ] );
  * @param  {object}  state   Global state tree
  * @param  {number}  siteId  Jetpack Site ID to fall back to
  * @param  {string | null}  themeId Theme ID
- * @returns {?Theme}         Theme object
+ * @returns {?CanonicalTheme}         Theme object
```

Comment:
I heard something about being possible to do:

```suggestion
 * @returns {?import('calypso/types').CanonicalTheme}         Theme object
```

So we can remove the useless import.

Also, what do you think of typing this file as well?

Reply:
I typed the file. I also found out that we're typing redux state in two or three different ways 😬 

1. `client/state/types.ts`
2. `client/types.ts:119`
3. Using `import { DefaultRootState } from 'react-redux';` like in `client/state/sites/selectors/get-site-products.ts`